.github/workflows/release_validate.yml

```diff
@@ -35,16 +35,22 @@ jobs:
 
       - uses: actions/setup-go@v5
```

Comment:
I guess there are two different things we want to do here:
* Check the release works with "whatever" version of client languages is going,
* (in a coming soon-PR) Rebuild the release and verify the checksum matches, if possible. (it is at least for Python, Node and Go).